### PR TITLE
chore(web): creating github corner icon

### PR DESCRIPTION
**Pull request by kawanofer**
Worked on the issue [25](https://github.com/brunocroh/bolhadev.chat/issues/25)

image

### CODE REVIEW

**Comment by brunocroh** [comment]
Nice, thank you for your contribution man, but we need some adjust before merge it

## Background is a gradient, so use this following colors to have exact same background
The exact tokens are:
 `#6171E5` 
 `#251B91` 
 
 But check on tailwind first if we have something similar to it, to prevent use of arbitrary values.

Screenshot 2024-06-03 at 09 09 17

## Adjust layout

In the original layout, we have something like a 'lighter' inner border
Screenshot 2024-06-03 at 09 36 54


## Here is the SVG used:

```svg
<svg width="49" height="48" viewBox="0 0 49 48" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M39.6908 5.61406C29.6524 -2.81606 14.7497 -1.57343 6.39462 8.38379C-0.212367 16.2577 -0.740731 27.2202 4.28242 35.6239C5.03716 36.6623 5.45565 35.8667 5.84111 35.4073L8.46159 32.2844C2.41376 28.3015 5.2268 24.4955 5.36448 23.581C5.67904 21.6887 4.18319 19.8842 5.21826 19.7227C7.0106 20.1348 7.61681 22.01 7.27642 25.4256C7.02634 27.9007 9.34071 29.3895 10.6203 30.0168C11.6464 29.2641 12.8518 28.8335 14.0851 28.7491C9.22803 22.9431 9.74965 17.489 13.1985 13.3788C14.8723 11.3841 17.0726 10.106 19.6129 9.71748C20.8302 6.58447 23.4872 5.25694 23.8694 5.04887C26.4044 6.81894 27.2746 10.7628 27.3403 11.072C28.9813 11.8378 30.671 12.9329 32.3846 14.3707C34.1063 15.8154 35.4755 17.3008 36.5234 18.7981C37.273 18.8365 40.9062 19.1247 42.9332 21.162C42.7906 21.5711 41.9649 24.3529 39.1667 26.1044C39.238 28.6913 38.356 31.1031 36.6789 33.1018C33.2232 37.22 27.923 38.6712 21.359 34.8664C21.4478 35.7565 21.3583 36.654 21.0956 37.5064C20.8329 38.3589 20.4024 39.1493 19.8291 39.8315L16.0252 44.3648C15.7534 44.7547 15.42 45.0861 16.0291 45.5972C25.239 49.2257 36.0731 46.8284 42.7375 38.886C51.096 28.9247 49.7245 14.0333 39.6908 5.61406Z" fill="white"/>
</svg>

```

## File names and component name
We are using kebab-case on file names, so a better name for this file is `github-corner.tsx` and component can be renamed to GithubCorner

## pnpm-lock.yaml

Do not commit this file, his changes are not related with PR itself.

## Git and branch naming

Use present tense on PR titles and commit messages as well

Inside of git files, they have a doc that describe how commits need to be written, and normally all projects follow it.
Screenshot 2024-06-03 at 09 28 46

[Source for you to read if you are interested](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?h=v2.36.1#n181)

## Conventional commits

As wrote on contributing guide, we are following conventional commits on this repo, so please follow it on your commits
https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?h=v2.36.1#n181

**Comment by brunocroh** [comment]
An excellent alternative is to use this one, using gradient as background and white Octocat

https://tholman.com/github-corners/#

File: .env.template
Comment (brunocroh) [inline .env.template:1]: Don't delete this file.

File: pnpm-lock.yaml
Comment (brunocroh) [inline pnpm-lock.yaml:1]: Remove thiese changes from your commit.

**Review by brunocroh — CHANGES_REQUESTED** [review]

**Comment by kawanofer** [comment]
Roger that. I'll make these changes. :)

**Comment by kawanofer** [comment]
@brunocroh do you have any other change suggestions I need to make?

**Review by brunocroh — APPROVED** [review]
Awesome work @kawanofer, thank you for your contribution

**Pull request merged**